pkg/resources/marketplace/manifestHelper.go

```diff
@@ -179,8 +178,8 @@ func CheckFoldersForMatch(dir string, folders []*semver.Version, currentFolder s
 }
 
 // GetCrdDetails reads the crd file
-func GetCrdDetails(crdConfig *apiextensionv1.CustomResourceDefinition, currentFolder string, f os.FileInfo) {
-	yamlFile, err := ioutil.ReadFile(filepath.Clean(currentFolder + string(os.PathSeparator) + f.Name()))
+func GetCrdDetails(crdConfig *apiextensionv1.CustomResourceDefinition, currentFolder string, f os.DirEntry) {
```

Comment:
Hi Kev. Do you know if this is covered by e2e testing? I don't see a related unit test.

Reply:
@laurafitzgerald Yes, I think this is generally covered by an installation using a config map catalog source (I believe marin3r installation is still using this instead of using an index). So I think as long as the e2e succeeds, this is technically covered. Installation will not complete if this change had any issues 🤔

Reply:
Thanks for clarification. In that case I'm happy to approve lgtm this if you are ready.

Reply:
Thanks ! Yeah, I think this PR is ready 👍